msg/vehicle_status.msg

```diff
@@ -80,3 +80,10 @@ uint8 failure_detector_status			# Bitmask containing FailureDetector status [0,
 uint32 onboard_control_sensors_present
 uint32 onboard_control_sensors_enabled
 uint32 onboard_control_sensors_health
+
+# airspeed fault and airspeed use status
+bool aspd_check_failing		# true when airspeed consistency checks are failing
+bool aspd_fault_declared	# true when an airspeed fault has been declared
+bool aspd_use_inhibit		# true if switching to a non-airspeed control mode has been requested
+bool aspd_fail_rtl		# true if airspeed failure invoked RTL has been requested
+float32 load_factor_ratio	# ratio of measured to aerodynamic load factor limit. Greater than 1 indicates airspeed low error condition.
```

Comment:
Perhaps this should be calculated somewhere centrally. The position controller is already doing similar things, but with the minimum airspeed instead of stall. Stall airspeed (COM_ASPD_STALL) could be defined along with MIN, MAX, TRIM.

Reply:
Possibly, but I am out of hours on this task and this type of restructure could take significant time.